From 0ffb543a01bdd32436079ed39472a62908ebe56f Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Wed, 02 Sep 2020 10:54:44 +0300
Subject: [PATCH] [SW-19433] make: hls: directly use Yocto layers

When using direct Yocto commands instead of buildroot targets, it is
required to run the rsync target between commands, if the Yocto layersd
have changes between runs. Directly sourcing the Yocto layers from
npu-stack removes that need, and thus makes development process far more
convenient.

Change-Id: I6db18b7e60150af2940f1f10ca6f71b3a972a0e2
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/scripts/make/hls/Makefile b/scripts/make/hls/Makefile
index e27d1c2..e2e570d 100644
--- a/scripts/make/hls/Makefile
+++ b/scripts/make/hls/Makefile
@@ -17,8 +17,8 @@
 export OPENBMC_BITBAKE_ENV
 
 define OPENBMC_ENV
-	mkdir -p $(OPENBMC_WORK_DIR);				\
-	TEMPLATECONF=$(OPENBMC_ENV_FLAGS) source ./openbmc-env;
+	TEMPLATECONF=$(OPENBMC_ENV_FLAGS) source ${HABANA_SOFTWARE_STACK}/embedded/openbmc/openbmc-env \
+		${OPENBMC_DEPLOY_DIR}/build;
 endef
 export OPENBMC_ENV
 
